docroot/themes/custom/uids_base/uids_base.libraries.yml

```diff
@@ -163,7 +163,7 @@ colorbox:
     name: MIT
     gpl-compatible: false
   js:
-    colorbox/jquery.colorbox-min.js: { minified: true }
+    ../../../libraries/colorbox/jquery.colorbox-min.js: { minified: true }
```

Comment:
I think you can use a docroot-relative path like we do for Chosen in the sitenow profile: https://github.com/uiowa/uiowa/blob/main/docroot/profiles/custom/sitenow/sitenow.libraries.yml#L18

Reply:
done in cf1ee54